workloads/large-messages-low-rate.sh

```diff
@@ -41,7 +41,7 @@ echo "Creating a NoOp destination..."
 DEST_CONN_REQ=$(
 jq -n  --arg pipeline_id "$PIPELINE_ID" '{
      "type": "TYPE_DESTINATION",
-     "plugin": "builtin:noopdest",
+     "plugin": "/plugins/conduit-connector-noop-dest",
```

Comment:
The plugin binary name could be _a bit_ shorter 😄

Reply:
Yeah, I initially wanted to have it consistent with other connectors. But given the very limited use for this destination, I can just call it noop-dest.